### DISCUSSION TITLE
Lazer Not on Linux Anymore?

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#30652] [Q&A] [answered]
Tried to install Lazer. Worked fine before the update, but now it says the download link no longer exists. And I also can't open my game anymore.

Was it removed by accident or something, or is this just a me thing?

![image](https://github.com/user-attachments/assets/879a31c7-73c4-42d9-9023-5ad399d15c02)
This is the screen I get when I try to install the game from the osu! website and from GitHub itself

**Comment by WalmartWhiteCat** [comment, marked as answer]
Nevermind, found what it is

**Reply by Bruno5430** [reply]
https://github.com/ppy/osu/issues/30648#issuecomment-2478931793

![image.png](https://imgs.xkcd.com/comics/wisdom_of_the_ancients.png)
